pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -72,6 +72,9 @@ spec:
     value: "default"
   - name: VolumeMonitor
     enabled: "true"
+  # ServiceAccountName is the account name assigned to volume management pod
```

Comment:
should the comment be updated to m-apiserver pod?

Reply:
Yes